src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    if (compareFunction === undefined) {
+      for (let i = 0; i < this.length - 1; i++) {
+        if (this[i] + '' > this[i + 1] + '') {
+          const currentIndex = this[i];
+
+          this[i] = this[i + 1];
+          this[i + 1] = currentIndex;
+
+          i = -1;
+        }
+      }
```

Comment:
The sorting algorithm used here is a variant of bubble sort, which is not efficient for large arrays. Also, the string conversion is not necessary for default sorting, as the default behavior is to convert to strings internally.